site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -41,6 +44,9 @@ A list of available out-of-the-box interactors is placed below.
 
   ![](/images/image224.jpg)
 
+- You can use the `Selected opacity` slider in the `Objects sidebar` to change the opacity of the polygon.
+  You can read more in the [Objects sidebar](/docs/manual/basics/objects-sidebar/#appearance) section.
+
```

Comment:
It also works for classic drawing modes and OpenCV intelligent scissors. Maybe this fact should be reflected in the documentation